CONTRIBUTING.md

```diff
@@ -41,10 +41,19 @@ git checkout -b feature/your-branch
 make go/test
 make test/e2e/<scope_of_the_changes>
 ```
+6. To test your changes on a cluster use
+* kubectl to connect to a cluster
+* Use make commands to build and deploy your operator as follows:
+```sh
+make build && make deploy
```

Comment:
`make images/build/push && make deploy`

Reply:
@wepudt  it's important to say: you have to push before deploy.

Reply:
isnt't that exactly what make build does?

from the Makefile:

`## Builds the operator image and pushes it to quay with a snapshot tag
build: images/build/push`

Reply:
yeah we have an alias on it:
```
## Builds the operator image and pushes it to quay with a snapshot tag
build: images/build/push
```
so confusing...